README.md

```diff
@@ -23,7 +23,7 @@ Shift continues to be immensely helpful for building the Coursera Android applic
 
 ####Gradle
 ``` java
-compile 'org.coursera.android:shift:1.0.1'
+compile 'org.coursera.android:shift:1.0.2'
```

Comment:
Was gonna make a separate commit for changing the version once this is merged and I reupload to jcenter but yea it should be 1.0.3.